### ISSUE TITLE
using mapbox public key doesn't work

### ISSUE THREAD

**Issue by jreiberkyle** [mapbox/mbview#79]
Following the instructions in the README, I set MAPBOX_ACCESS_KEY to the public key given. When I attempted to view the example mbtiles, I was met with a blank page. When I switched to using my private access key, the mbtiles were loaded. Is this issue specific to me or is the README outdated?

My steps using the public key:
```
npm install -g mbview
git clone git@github.com:mapbox/mbview
cd mbview/examples
export MAPBOX_ACCESS_TOKEN='pk.0000.1111'
mbview baja-highways.mbtiles 
```

**Comment by andrewharvey** [comment]
@jreiberkyle Did you replace `pk.0000.1111` with a public access token from your account at https://www.mapbox.com/account/access-tokens?

`pk.0000.1111` is just an example it's not a valid access token, you need to use a public access token from your own account.

**Comment by jreiberkyle** [comment]
@andrewharvey yep, and that worked!

I didn't realize from the documentation that `pk.0000.1111` is just an example and that mbtiles requires my public access token to work. Part of the issue may be that it's not intuitive to me that a token would be required to view my own local mbtiles.

**Comment by andrewharvey** [comment]
@jreiberkyle It's because mbview uses a Mapbox basemap as a background to your own local mbtiles. I'm think a PR to make that part optional, and hence work without a Mapbox access token would be welcomed.

**Comment by jreiberkyle** [comment]
@andrewharvey that is great and I'm sure that patch would be useful in that it would lower the barrier to using mbview! I took a quick look at the code and it looks like the style used to render vector features also requires a mapbox token. For now, I have created a PR (above, #80) that hopefully clarifies the need for a mapbox public token.

**Comment by mollymerp** [comment]
fixed in #80
